test/binCases/help/mixed/mixed.test.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
```

Comment:
All the tests are about the help command. Why did you decide to create a separate file for each case instead to have one file with different cases there?

Reply:
probably because they tend to get very big